src/network.js

```diff
@@ -310,7 +310,9 @@ export class BlockstackNetwork {
    * @return {Promise} a promise that resolves to the WHOIS-like information 
    */
   getNameInfo(fullyQualifiedName: string) {
-    return fetch(`${this.blockstackAPIUrl}/v1/names/${fullyQualifiedName}`)
+    console.log(this.blockstackAPIUrl)
```

Comment:
this should be a debug statement

Reply:
Thanks for catching this! I pushed a fixed.